### ISSUE TITLE
mermaid does not shows class diagram

### ISSUE THREAD

**Issue by Noobzik** [benweet/stackedit#1587]
I hve tested this following code into the online editor and mermaid does not recognize it. Maybe updating the plugin should resolve the issue ?
```
\```mermaid
classDiagram
class Serie
  Serie : serieID
  Serie : titre
  Serie : annee
\```
```

**Comment by floriama** [comment]
Same issue for me. Would love to be able to use this in stackedit!

**Comment by sanjayabc1234** [comment]
Yes, the class diagrams are not working. It would be great to have them. I tried this:

```mermaid
classDiagram
    class Animal
    Vehicle <|-- Car
```
Thanks for awesome application.

**Comment by benweet** [comment]
Fixed with v5.14.10.
